x-pack/legacy/plugins/siem/public/containers/case/use_get_case.tsx

```diff
@@ -75,7 +77,7 @@ export const useGetCase = (caseId: string): [CaseState] => {
     const fetchData = async () => {
       dispatch({ type: FETCH_INIT });
       try {
-        const response = await getCase(caseId, false);
+        const response = await getCase(caseId, true);
```

Comment:
FMI -> the true or false is to get the comments with the case if that's true maybe we can add a parameter and default to true in the hook function. if we need later on to just get the case without comment.